packages/web-frontend/src/containers/EnlargedDialog/EnlargedDialogContent.js

```diff
@@ -104,23 +104,21 @@ export class EnlargedDialogContent extends PureComponent {
       return null;
     }
 
-    const { name, periodGranularity } = viewContent;
+    const { name, periodGranularity, reference } = viewContent;
 
     if (viewContent.entityHeader === '') titleText = `${viewContent.name}`;
     else if (viewContent.entityHeader)
       titleText = `${viewContent.name}, ${viewContent.entityHeader}`;
     else titleText = `${name}${organisationUnitName ? `, ${organisationUnitName} ` : ''}`;
 
-    const style = {
-      textAlign: 'center',
-      color: isExporting ? DARK_BLUE : WHITE,
-    };
-
     return (
-      <DialogTitle style={style}>
-        <span style={styles.titleText}>{titleText}</span>
```

Comment:
perhaps also remove the titleText style in this file since it's no longer needed here?